app/Resources/views/receipt/create_receipt.html.twig

```diff
@@ -21,9 +21,13 @@
 
         <div class="col-12 col-sm-6 d-flex flex-column">
             <div class="{{ receipt.picturePath is not null ? '' : 'd-none' }}">
-                <img class="img-fluid" src="{{ asset(receipt.picturePath) }}" id="receipt_picture">
+				<img class="img-fluid d-none" src="{{ asset(receipt.picturePath) }}" id="receipt_picture">
+				<object id="receipt_pdf" data="{{asset(receipt.picturePath)}}" class="d-none" style="max-width: 100%; max-height:70%;" type="application/pdf" width="400" height="600">
+					<embed src="{{asset(receipt.picturePath)}}" type="application/pdf">
+						<p>This browser does not support PDFs. Please download the PDF to view it: <a href="{{asset(receipt.picturePath)}}">Download PDF</a>.</p>
```

Comment:
All other text shown to the user is in Norwegian. 
The feedback should be as well for consistency.